drivers/dai/intel/uaol/uaol-params-intel-ipc4.h

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2023-2024 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __INTEL_DAI_DRIVER_UAOL_PARAMS_IPC4_H__
+#define __INTEL_DAI_DRIVER_UAOL_PARAMS_IPC4_H__
```

Comment:
Why not `INTEL_DAI_DRIVER_UAOL_PARAMS_IPC4_H_`?

Reply:
Fixed.